rails/benchmarks/bm_activerecord_scope_to_a.rb

```diff
@@ -0,0 +1,32 @@
+require 'bundler/setup'
+require 'active_record'
+require_relative 'support/benchmark_rails'
+
+ActiveRecord::Base.establish_connection(ENV['DATABASE_URL'])
+
+ActiveRecord::Migration.verbose = false
+
+ActiveRecord::Schema.define do
+  create_table :users, force: true do |t|
+    t.string :name, :email
+    t.timestamps null: false
+    t.index(:email, unique: true)
```

Comment:
Just a nitpick here. Lets leave out the parenthesis for consistency with the lines above.